components/ui-shell/settings.ts

```diff
@@ -88,7 +88,11 @@ export const NAV_ITEMS: TopLevelNavItem[] = [
         label: 'Getting Started',
```

Comment:
```suggestion
        label: 'Start Here',
```

Can you change this to `Start Here` to avoid confusion with the getting started link below?